plugins/modules/rundeck_acl_policy.py

```diff
@@ -204,6 +210,7 @@ def main():
     argument_spec.update(dict(
         state=dict(type='str', choices=['present', 'absent'], default='present'),
         name=dict(required=True, type='str'),
+        api_token=dict(required=True, type="str", no_log=True, aliases=["token"]),
```

Comment:
Same here: I would simply add `argument_spec['api_token']['aliases'] = ['token']` after the `update()` instead of adding it here.